examples/get-started/pure-js/widgets/app.js

```diff
@@ -0,0 +1,85 @@
+import {Deck} from '@deck.gl/core';
+import {GeoJsonLayer, ArcLayer} from '@deck.gl/layers';
+import {FullscreenWidget} from '@deck.gl/widgets';
+import '@deck.gl/widgets/stylesheet.css';
+import {luma} from '@luma.gl/api';
+import {WebGLDevice} from '@luma.gl/webgl';
+
+luma.registerDevices([WebGLDevice]);
+
+const prefersDarkScheme = window.matchMedia("(prefers-color-scheme: dark)");
+const widgetTheme = prefersDarkScheme.matches ? {
+  // Dark Glass Theme
+  '--button-background': 'rgba(18, 18, 20, 0.75)',
+  '--button-stroke': 'rgba(18, 18, 20, 0.30)',
+  '--button-inner-stroke': '1px solid rgba(18, 18, 20, 0.75)',
+  '--button-shadow': '0px 0px 8px 0px rgba(0, 0, 0, 0.25), 0px 0px 8px 0px rgba(0, 0, 0, 0.1) inset',
+  '--button-backdrop-filter': 'blur(4px)'
+} : {
```

Comment:
Demonstrates how an application can set the theme from JS.

Reply:
OK but the default dark and light themes should not be in the app right, they should be provided by the widgets module? Maybe less confusing if the app was setting a unique theme (say a game UI theme or a theme that favors red color etc)

Reply:
I'd prefer to keep the widget's "dumb" in the sense that they won't try to automatically apply a theme based on mobile or dark/light OS theme. Reason being, apps will already own their approach on when to apply these responsive styles and there are potentially many edge cases and oddities to cover. 

For instance, apps may have a custom user-override for light/dark, they may want a different size for iPad, etc..

As long as they can apply a theme in their CSS or via JS, they should be good.

Reply:
Yews but even if the widgets don't do any auto switching of themes,  the app shouldn't have to define the full themes?

I was assuming that the "themes" should be importable from the widget module.

Seeing that an example app has to tediously itemize detailed sets of CSS values does not give me a sense that I get easy theming with this widget library. 

```
import {LightTheme, DarkTheme} from '@deck.gl/widgets';
const widgetTheme = prefersDarkScheme.matches ? LightTheme : DarkTheme;
```

Reply:
Oh, good idea. Then users could import a theme and modify maybe one or two variables as-needed too.